src/content/structured/components/footer/code.mdx

```diff
@@ -213,17 +234,81 @@ export const snippetsLogoLinks = [
   <IcFooterLink slot="link" href="/">Patterns</IcFooterLink>
   <div slot="logo">
     <IcFooterLink href="/">
-      <svg ...></svg>
+      <SlottedSVG xmlns="http://www.w3.org/2000/svg"
+        height="48px"
+        viewBox="0 0 24 24"
+        width="48px"
+        fill="#FFFFFF"
+      >
+        <path
+          d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+          fill="currentColor"
+        />
+      </SlottedSVG>
     </IcFooterLink>
     <IcFooterLink href="/">
-      <svg ...></svg>
+      <SlottedSVG xmlns="http://www.w3.org/2000/svg"
+        height="48px"
+        viewBox="0 0 24 24"
+        width="48px"
+        fill="#FFFFFF"
+      >
+        <path
+          d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+          fill="currentColor"
+      />
+      </SlottedSVG>
     </IcFooterLink>
   </div>
 </IcFooter>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsLogoLinks}>
+  <IcFooter description="The ICDS is maintained by the Design Practice Team. If you've got a question or want to feedback, please get in touch.">
```

Comment:
I think all of this code is repeated below, it should only need the svgs updating